src/dispatch/dispatcher.rs

```diff
@@ -151,8 +131,8 @@ impl<'a, 'b, 'c> RunNow<'a> for Dispatcher<'b, 'c> {
         self.setup(world);
     }
 
-    fn dispose(&mut self, world: &mut World) {
-        self.dispose(world);
+    fn dispose(self: Box<Self>, world: &mut World) {
```

Comment:
will this bump min rust version?

Reply:
No, it won't, that's stable since 1.2x and we're now on 1.32.

Reply:
Also, that's ensured by CI.

Reply:
huh I thought self-referential types was like 1.3X 🤷‍♂️

Reply:
That was just for things **other than `Box`**